### ISSUE TITLE
Error after installing into virtualenv

### ISSUE THREAD

**Issue by alex** [j-bennet/dockercli#83]
```
(tempenv-584b169716017) ~ $ pip install dockercli
Collecting dockercli
Collecting configobj>=5.0.6 (from dockercli)
Collecting pexpect>=3.3 (from dockercli)
Collecting docker-py>=1.2.0 (from dockercli)
Collecting fuzzyfinder>=1.0.0 (from dockercli)
  Using cached fuzzyfinder-1.0.0-py2.py3-none-any.whl
Collecting click>=4.0 (from dockercli)
  Using cached click-4.1-py2.py3-none-any.whl
Collecting tabulate>=0.7.5 (from dockercli)
Collecting py-pretty>-0.1 (from dockercli)
Collecting prompt-toolkit==0.38 (from dockercli)
  Using cached prompt_toolkit-0.38-py2-none-any.whl
Collecting pygments>=2.0.2 (from dockercli)
  Using cached Pygments-2.0.2-py2-none-any.whl
Collecting six (from configobj>=5.0.6->dockercli)
  Using cached six-1.9.0-py2.py3-none-any.whl
Collecting websocket-client>=0.32.0 (from docker-py>=1.2.0->dockercli)
Collecting requests>=2.5.2 (from docker-py>=1.2.0->dockercli)
  Using cached requests-2.7.0-py2.py3-none-any.whl
Collecting wcwidth (from prompt-toolkit==0.38->dockercli)
  Using cached wcwidth-0.1.4-py2.py3-none-any.whl
Installing collected packages: six, configobj, pexpect, websocket-client, requests, docker-py, fuzzyfinder, click, tabulate, py-pretty, pygments, wcwidth, prompt-toolkit, dockercli
Successfully installed click-4.1 configobj-5.0.6 docker-py-1.3.0 dockercli-0.3 fuzzyfinder-1.0.0 pexpect-3.3 prompt-toolkit-0.38 py-pretty-1 pygments-2.0.2 requests-2.7.0 six-1.9.0 tabulate-0.7.5 wcwidth-0.1.4 websocket-client-0.32.0
(tempenv-584b169716017) ~ $ dockercli
Traceback (most recent call last):
  File "/Users/alex_gaynor/.virtualenvs/tempenv-584b169716017/bin/dockercli", line 11, in <module>
    sys.exit(cli())
  File "/Users/alex_gaynor/.virtualenvs/tempenv-584b169716017/lib/python2.7/site-packages/click/core.py", line 664, in __call__
    return self.main(*args, **kwargs)
  File "/Users/alex_gaynor/.virtualenvs/tempenv-584b169716017/lib/python2.7/site-packages/click/core.py", line 644, in main
    rv = self.invoke(ctx)
  File "/Users/alex_gaynor/.virtualenvs/tempenv-584b169716017/lib/python2.7/site-packages/click/core.py", line 837, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/alex_gaynor/.virtualenvs/tempenv-584b169716017/lib/python2.7/site-packages/click/core.py", line 464, in invoke
    return callback(*args, **kwargs)
  File "/Users/alex_gaynor/.virtualenvs/tempenv-584b169716017/lib/python2.7/site-packages/dockercli/main.py", line 316, in cli
    dockercli = DockerCli()
  File "/Users/alex_gaynor/.virtualenvs/tempenv-584b169716017/lib/python2.7/site-packages/dockercli/main.py", line 57, in __init__
    self.config = self.read_configuration()
  File "/Users/alex_gaynor/.virtualenvs/tempenv-584b169716017/lib/python2.7/site-packages/dockercli/main.py", line 76, in read_configuration
    write_default_config(default_config, self.config_name)
  File "/Users/alex_gaynor/.virtualenvs/tempenv-584b169716017/lib/python2.7/site-packages/dockercli/config.py", line 37, in write_default_config
    shutil.copyfile(source, destination)
  File "/Users/alex_gaynor/.pyenv/versions/2.7.10/lib/python2.7/shutil.py", line 82, in copyfile
    with open(src, 'rb') as fsrc:
IOError: [Errno 2] No such file or directory: u'/Users/alex_gaynor/.virtualenvs/tempenv-584b169716017/lib/python2.7/site-packages/dockercli/dockerclirc'
```

**Comment by greyhoundforty** [comment]
I have the same error on a fresh install on Ubuntu 14:

```
root@arcadia:~# dockercli
Traceback (most recent call last):
  File "/usr/local/bin/dockercli", line 9, in <module>
    load_entry_point('dockercli==0.3', 'console_scripts', 'dockercli')()
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 664, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 644, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 837, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 464, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/dockercli/main.py", line 316, in cli
    dockercli = DockerCli()
  File "/usr/local/lib/python2.7/dist-packages/dockercli/main.py", line 57, in __init__
    self.config = self.read_configuration()
  File "/usr/local/lib/python2.7/dist-packages/dockercli/main.py", line 76, in read_configuration
    write_default_config(default_config, self.config_name)
  File "/usr/local/lib/python2.7/dist-packages/dockercli/config.py", line 37, in write_default_config
    shutil.copyfile(source, destination)
  File "/usr/lib/python2.7/shutil.py", line 82, in copyfile
    with open(src, 'rb') as fsrc:
IOError: [Errno 2] No such file or directory: u'/usr/local/lib/python2.7/dist-packages/dockercli/dockerclirc'
```

**Comment by j-bennet** [comment]
Thanks for reporting, will investigate asap.

**Comment by TheHippo** [comment]
Get the same error when installing without `virutalenv`.

Ubuntu 14.04. Python wise everything as it ships with the Ubuntu repositories.

**Comment by j-bennet** [comment]
Fixed in v0.6 that's now on pypi. Thanks @amjith for the fix, and everyone else for trying out and reporting.
